src/construction.cpp

```diff
@@ -157,9 +157,12 @@ static bool check_nothing( const tripoint_bub_ms & )
 static bool check_channel( const tripoint_bub_ms & ); // tile has adjacent flowing water
```

Comment:
Now you've properly documented the post specials it might make sense to delete the C++ pre/post special description comments so there's not 2 lists that are likely to become outdated compared to each other

Reply:
I've slept on your comment, and unfortunately I disagree. The reason is that I'm not trusting documentation fully as it gets outdated, and the source, in my view, is the actual code. Note that comments in the code shouldn't be trusted absolutely either: they tend to state what the writer intended the code to do at some time, but may not have been updated either, or the author may have bungled the code so it didn't actually do what was intended.

Thus, my truth ladder is:
1. The code (or, actually, the generated assembly, as compiler bugs exist. Fortunately it's very rare to need to go to the assembly these days).
2. Comments in the code, acting as guides.
3. Documentation.

Note, however, that the "find out info" sequence is typically in the reverse order, because documentation tends to try to require less prerequisites and provide a high level picture.

Reply:
That's reasonable, my issue with the duped lists is that anyone changing the code is likely to change the code comments given that they're right in their face and then may well leave it at that, leading to the docs becoming outdated and causing your truth ladder whereas linking to the docs puts that in their face instead hopefully encouraging them to go change that instead.